### PR TITLE
fix(logs): BS-15073: set better default log level + log format

### PR DESCRIPTION
**Pull request by educhastenier**

### CODE REVIEW

File: deploy/distrib/src/main/resources/tomcat/conf/logging.properties [hunk 1]
```diff
@@ -23,7 +23,9 @@ handlers = java.util.logging.ConsoleHandler, 1catalina.org.apache.juli.FileHandl
 .handlers = 1catalina.org.apache.juli.FileHandler, java.util.logging.ConsoleHandler
 
 # Allow logging to trace the actual logger producer class:
-java.util.logging.SimpleFormatter.format=%1$tF %1$tT.%1$tL %1$tz %3$s %2$s %n%4$s: %5$s%6$s%n
+java.util.logging.SimpleFormatter.format=%1$tF %1$tT.%1$tL %1$tz %4$s: %3$s %5$s%6$s%n
```
Comment (laurentleseigneur) [inline deploy/distrib/src/main/resources/tomcat/conf/logging.properties]: an example of before / after could be more relevant to explain the benefit of this change

Comment (educhastenier) [inline deploy/distrib/src/main/resources/tomcat/conf/logging.properties]: before:
2016-05-11 14:44:30.352 +0200 org.bonitasoft.platform.configuration.util.AllConfigurationResourceVisitor org.bonitasoft.platform.configuration.util.AllConfigurationResourceVisitor visitFile 
INFOS: found file: tenant_template_engine/bonita-tenants-custom.xml
after:
2016-05-11 12:09:10.201 +0200 INFOS: org.bonitasoft.platform.configuration.util.AllConfigurationResourceVisitor found file: tenant_template_engine/bonita-tenants-custom.xml

**Comment by laurentleseigneur** [comment]
missing JBOSS config

**Comment by laurentleseigneur** [comment]
this change should be mentionned in release note IMHO

File: deploy/distrib/src/main/resources/tomcat/conf/logging.properties (hunk 1 above)
Comment (educhastenier) [inline deploy/distrib/src/main/resources/tomcat/conf/logging.properties]: done

**Comment by educhastenier** [comment]
* JBoss config done
* Release notes updated

**Comment by educhastenier** [comment]
Then, you can merge https://github.com/bonitasoft/bonita-distrib-sp/pull/24

**Comment by laurentleseigneur** [comment]
new logging format does not appear in jboss xml config file ? is it hidden somewhere else ?

**Comment by laurentleseigneur** [comment]
LGTM

**Pull request merged**